app/assets/stylesheets/article-show.scss

```diff
@@ -58,10 +58,14 @@ article {
     width: 100%;
     margin: auto;
     max-width: 1024px;
-    background: transparent no-repeat center center;
-    background-size: cover;
     z-index: 2;
-    padding-top: 42%;
+    background: transparent no-repeat center center; /* TODO[team-delightful]: Remove background styling a few days after merge to avoid conflicts */
```

Comment:
GREAT use of notes-to-self here :) Might want to just move this up to the top of the class definition and explicitly note that we can do this for both `background` and `background-cover`

Reply:
Also, I'd recommend creating an issue for this, and linking to it here. That way there will be more context for anyone who touches this, and you can track it with an issue! But this is just my personal workflow/style, so definitely not blocking.

Reply:
Thank you! 😄 

In addition to making the comment slightly more explicit and moving it to the top of the class, I think opening an issue for this TODO is an excellent idea--I'm on it. 👍 👍